xbmc/video/VideoDatabase.cpp

```diff
@@ -419,13 +419,17 @@ void CVideoDatabase::CreateViews()
                                      "  LEFT JOIN rating ON"
                                      "    rating.rating_id=tvshow.c%02d "
                                      "  LEFT JOIN uniqueid ON"
-                                     "    uniqueid.uniqueid_id=tvshow.c%02d "
-                                     "GROUP BY tvshow.idShow",
```

Comment:
I know I'm quite late but this might cause duplicated entries. The tvshowlinkpath is there actually because you might have more than one path for a TV show..this means that without grouping you'll get a duplicated TV show from the query

Reply:
Hi, as the tvshowlinkpath is not within an aggregate function, then even with group by  (or with SELECT DISTINCT) different tvshowlinkpath values will be selected in a different row. Does it really have to be in there? Which functionality is using this view?